kolibri/plugins/device/assets/src/views/SelectContentPage/ChannelContentsSummary.vue

```diff
@@ -12,23 +12,32 @@
           {{ $tr('unlistedChannelTooltip') }}
         </KTooltip>
         <h1>
-          <KLabeledIcon icon="channel" :label="channel.name">
+          <KLabeledIcon icon="channel">
```

Comment:
Could you elaborate on your concerns about using the default slot instead of the `label` prop and the documentation? The default slot is supported and documented in KDS already https://design-system.learningequality.org/klabeledicon/#slot:default

> Optional slot as alternative to label prop

Or is your question rather about the text truncation?

Reply:
Oh, you are right - somehow I missed that!